test/extended/oauth/server_headers.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/openshift/origin/test/extended/util/oauthserver"
 )
 
-var _ = g.Describe("[sig-auth][Feature:OAuthServer] [Headers]", func() {
+var _ = g.Describe("[sig-auth][Feature:OAuthServer] [Headers][apigroup:route.openshift.io]", func() {
```

Comment:
This is going to fail for something else.  We can deal with it later, but this is going to fail because we have no oauth server.  I suspect we'll need a detection in the test based on whether we have the well_known.